### ISSUE TITLE
Not handling composite Primary Keys correctly

### ISSUE THREAD

**Issue by ShaunParsons** [sequelize/sequelize-auto#223]
## Table Definition
```psql
CREATE TABLE auth.role (
    tenant_id uuid NOT NULL,
    id uuid NOT NULL DEFAULT crypt.gen_random_uuid(),
    name text NOT NULL,

    PRIMARY KEY (tenant_id, id),
    UNIQUE (id),
    UNIQUE (tenant_id, name),
    FOREIGN KEY (tenant_id) REFERENCES auth.tenant (id)
);
```

## Expected Output

```js
/* jshint indent: 1 */

module.exports = function(sequelize, DataTypes) {
	return sequelize.define('role', {
		tenantId: {
			type: DataTypes.UUIDV4,
			allowNull: false,
			primaryKey: true,
			references: {
				model: 'tenant',
				key: 'id'
			},
			unique: true,
			field: 'tenant_id'
		},
		id: {
			type: DataTypes.UUIDV4,
			allowNull: false,
			defaultValue: sequelize.fn('crypt.gen_random_uuid'),
			unique: true,
			primaryKey: true,
			field: 'id'
		},
		name: {
			type: DataTypes.TEXT,
			allowNull: false,
			field: 'name'
		}
	}, {
		tableName: 'role',
		schema: 'auth'
	});
};
```

## Actual Output

```js
/* jshint indent: 1 */

module.exports = function(sequelize, DataTypes) {
	return sequelize.define('role', {
		tenantId: {
			type: DataTypes.UUIDV4,
			allowNull: false,
			primaryKey: true,
			references: {
				model: 'tenant',
				key: 'id'
			},
			unique: true,
			field: 'tenant_id'
		},
		id: {
			type: DataTypes.UUIDV4,
			allowNull: false,
			defaultValue: sequelize.fn('crypt.gen_random_uuid'),
			unique: true,
			field: 'id'
		},
		name: {
			type: DataTypes.TEXT,
			allowNull: false,
			primaryKey: true,
			field: 'name'
		}
	}, {
		tableName: 'role',
		schema: 'auth'
	});
};
```

For some reason its not detecting `id` as part of the composite Primary Key, but thinks `name` is part of it. I've got as far as https://github.com/sequelize/sequelize-auto/blob/master/lib/index.js#L221 and I _might_ be able to raise a PR, but any insight would be greatly appreciated.

EDIT:

This is based on Postgres 10

**Comment by ShaunParsons** [comment]
So found that the PRIMARY KEY is calculated from within sequelize itsself: https://github.com/sequelize/sequelize/blob/v3.30.2/lib/dialects/postgres/query-generator.js#L120

This query is missing a `constraint_name` from the JOIN between `table_constraints` & `key_column_usage`. This does seem to be fixed in the latest version of sequelize: https://github.com/sequelize/sequelize/blob/master/lib/dialects/postgres/query-generator.js#L114

**Comment by steveschmitt** [comment]
I think this is fixed in `sequelize-auto@0.5.0`  
Please try it and close this issue if so.  Thanks!

**Comment by steveschmitt** [comment]
Please try in `sequelize-auto@0.5.1`.
